ui/src/testing/mock-google-module.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+class LatLng {
+  constructor(lat, lng) {
+    this.lat = lat;
+    this.lng = lng;
+  }
+}
+
+class Bounds {
+  constructor() {
+    // TODO(nworden): see if there's a better way to do this. seems like there
```

Comment:
did you mean to leave this unresolved?

Reply:
Yeah, for now. It's only in this file so I don't think it's PR-blocking.